lib/membrane/rtp/jitter_buffer.ex

```diff
@@ -146,4 +161,75 @@ defmodule Membrane.RTP.JitterBuffer do
 
   defp record_to_action(nil), do: {:event, {:output, %Membrane.Event.Discontinuity{}}}
   defp record_to_action(%Record{buffer: buffer}), do: {:buffer, {:output, buffer}}
+
+  # TODO: stats spec could be improved
+  @spec get_and_update_stats(State.t()) :: {stats, State.t()} when stats: map()
+  def get_and_update_stats(%State{
```

Comment:
- `update_and_get` is even worse as the values for stats are generated (`get`) and `update` is done based on these stats. However, both suck, this could be something like `read_and_reset` as storing expected and received effectively resets the counter for `fraction_lost`
- Actually, it's going to become private and will be called after receiving a message requesting stats from 

Anyway, I'll add this as TODOs and fix in next PR as I've already started working on code using this and don't want to introduce conflicts